cmd/dagger/module.go

```diff
@@ -877,7 +877,7 @@ func optionalModCmdWrapper(
 			switch {
 			case configExists:
 				mod := modSrc.AsModule()
-				err := mod.Serve(ctx)
+				err := mod.Serve(ctx, dagger.ModuleServeOpts{IncludeDependencies: true})
```

Comment:
This makes `dagger listen` and `dagger query` behave the same by serving dependencies too, so it's more consistent. It's also helpful to debug what the shell sees.